src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/DeviceJobs/GetAzureStorSimpleJob.cs

```
@@ -0,0 +1,190 @@
+
+                if (ParameterSetName == StorSimpleCmdletParameterSet.IdentifyById)
+                {
+                    if (response == null || response.DeviceJobList.Count < 1)
+                    {
+                        WriteVerbose(string.Format(Resources.NoDeviceJobFoundWithGivenIdMessage, JobId));
+                        WriteObject(null);
+                        return;
+                    }
+                    WriteObject(response.DeviceJobList.First());
+                    return;
+                }
+                else
+                {
+                    WriteObject(response.DeviceJobList, true);
+                    WriteVerbose(string.Format(Resources.DeviceJobsReturnedCount, response.DeviceJobList.Count));
+                    if (response.NextPageUri != null
+                        && response.NextPageStartIdentifier != "1")
+                    {
+                        if (First != null)
+                        {
+                            //user has provided First(Top) parameter while calling the commandlet
+                            //so we need to provide it to him for calling the next page
+                            WriteVerbose(string.Format(Resources.DeviceJobsNextPageFormatMessage, First, response.NextPageStartIdentifier));
+                        }
+                        else
+                        {
+                            //user has NOT provided First(Top) parameter while calling the commandlet
+                            //so we DONT need to provide it to him for calling the next page
+                            WriteVerbose(string.Format(Resources.DeviceJobsNextPagewithNoFirstMessage, response.NextPageStartIdentifier));
+                        }
+                    }
+                    else
+                    {
+                        WriteVerbose(Resources.DeviceJobsNoMorePagesMessage);
+                    }
+                }
+            }
+            catch (Exception exception)
```

why is the exception being ignored?